graphql-dgs-spring-graphql/build.gradle.kts

```diff
@@ -18,6 +18,7 @@
 dependencies {
     implementation(project(":graphql-dgs"))
     implementation(project(":graphql-dgs-reactive"))
+    implementation(project(":graphql-dgs-spring-boot-oss-autoconfigure"))
```

Comment:
Yes that can happen, thanks! In this case, the dependencies change is related and on purpose. In this change, I added the configuration option in `graphql-dgs-spring-boot-oss-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/autoconfig/DgsConfigurationProperties.kt` which was in `graphql-dgs-spring-boot-oss-autoconfigure` package. 

When I tested in spring-graphql integration I needed to utilize that config property and In order to to use it in `graphql-dgs-spring-graphql/src/main/kotlin/com/netflix/graphql/dgs/springgraphql/autoconfig/DgsSpringGraphQLSourceAutoConfiguration.kt`  in `graphql-dgs-spring-graphql` module I needed to expand the scope from testImplementation to implementation scope. That is why this change is in dependencies was made.

Reply:
Note that for Netflix internal use we override that autoconfiguration module, so you would have to duplicate it there. The reason for that duplication is historical, and happened when we initially open sourced. We couldn't fix that without breaking users, but it's not ideal. Adding this dependency also creates an issue there.

I'm wondering if we would be better off only adding this for the spring-graphql version - that way it can be in the spring-graphql module and there's no duplication. @srinivasankavitha WDYT?

Reply:
More as a general point - anytime something changes with adding/removing dependencies you should call it out explicitly. It's easy to miss in a PR, but it can have pretty serious and hard to reverse effects.

Reply:
Yeah I agree, this can be supported only in the spring-graphql version. No need to backport new features in the non-spring graphql DGS, I agree. We will anyway deprecate that soon.

Reply:
Removed backport